core/queries/GetArticles.js

```diff
@@ -0,0 +1,13 @@
+import gql from "graphql-tag";
+
+export default gql`
+  {
```

Comment:
Missing operation name for this query
`query GetArticles....`
This applies for all queries.